server/data/data-file/src/test/java/org/apache/james/mailrepository/MBoxMailRepositoryTest.java

```diff
@@ -37,7 +37,7 @@
  * NOTE this previously extended AbstractMailRepositoryTest to run all of the
  * common mail repository tests on the MBox implementation.
  *
- * @Depracted: See JAMES-2323
+ * @Deprecated: See JAMES-2323
```

Comment:
why sometimes 2571 and sometimes 2323?

Reply:
Because the copy/paste was from an old commit which have the typo...